src/firejail/main.c

```diff
@@ -1025,7 +1029,7 @@ int main(int argc, char **argv, char **envp) {
 
 	// sanity check for environment variables
 	if (i >= MAX_ENVS) {
-		fprintf(stderr, "Error: too many environment variables\n");
+		fprintf(stderr, "Error: too many environment variables: >= MAX_ENVS (%d)\n", MAX_ENVS);
```

Comment:
```suggestion
		fprintf(stderr, "Error: too many environment variables: %d >= MAX_ENVS (%d)\n", i, MAX_ENVS);
```

Print the current amount of env vars.

Reply:
Well, I wanted to, but: i is not actually the amount of env vars. It is only how far we have counted up so far. The for loop above quits once i reaches our limit of MAX_ENVS. So if we print i here, we are not actually printing any additional information.

Reply:
Sorry, I misread the code.

Nevermind this suggestion.